src/main.rs

```diff
@@ -25,13 +26,33 @@ fn main() {
                     .expect("Pass a path after the '--path' flag")
                     .to_string()
             }
+            "--header" => {
+                let mut new_header = args
+                    .next()
+                    .expect("Pass a header after the '--header' flag")
+                    .to_string();
+                if !new_header.contains(':') {
+                    if new_header.contains('=') {
+                        new_header = new_header.replacen("=", ":", 1);
+                    } else {
+                        panic!("Pass a ':' or '=' in the '--header' flag");
+                    }
+                }
+                if new_header.contains('\r') || new_header.contains('\n') || !new_header.is_ascii()
+                {
+                    panic!("Only ASCII without line breaks is allowed in the '--header' flag");
+                }
+                headers.push_str("\r\n");
+                headers.push_str(&new_header);
+            }
             "--help" => {
                 println!(
                     r#"Run 'devserver' in a folder to host that folder.
 
 --reload                   Automatically refresh pages when a file in the hosted folder changes. Disabled by default.
 --address [address]:[port] Specify an address to use. The default is 'localhost:8080'.
 --path [path]              Specify the path of the folder to be hosted.
+--header                   Specify an additional header to send in responses.
 --help                     Display the helpful information you're reading right now.
 
 Examples:
```

Comment:
The example should be extended so people can see how `--header` flag is formatted.

`devserver --address 127.0.0.1:8080 --path "some_directory/subdirectory" --header Access-Control-Allow-Origin='*'`

Reply:
Thanks! Done in ab728ee7997e43c8a77291f488a273b1dfb0a8ff as well.